content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst

```diff
@@ -0,0 +1,217 @@
+============
+Product type
+============
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+Define *product types* in Odoo to determine how the database keeps track of the product.
+
+.. seealso::
+   `Odoo Tutorials: Product Type <https://www.youtube.com/watch?v=l6j0ZkP5mLM>`_
+
+Set product type
+================
+
+To set a product type, go to to :menuselection:`Inventory app --> Products --> Product` and
+selecting the desired product.
+
+On the product form, in the :guilabel:`Product Type` field, select:
+
+- :guilabel:`Storable` for products tracked with stock counts. Only storable products can trigger
+  reordering rules for generating purchase orders.
+
+.. tip::
+   Choose :guilabel:`Storable` if it is necessary to track a product's stock at various locations,
+   inventory valuation, or lots and serial numbers.
+
+- :guilabel:`Consumable` are items always assumed to be in stock, but with untracked and
+  unforecasted quantities. Examples include nails, toilet paper, and coffee—replaceable, essential,
+  but exact counts are unnecessary
+- :guilabel:`Service` are products are services performed by the business that is for sale (e.g.
+  maintenance, installation, repair)
+
+.. image:: type/product-form.png
+   :align: center
+   :alt: Set a product type on the product form.
+
+.. note::
+   The product types listed above are part of the standard *Inventory* app. For access to the fields
+   below, install the corresponding apps **in addition** to *Inventory*.
+
+   To install additional applications, navigate to :menuselection:`Apps` and select the
+   :guilabel:`Activate` button to install the desired apps or modules.
+
+- :guilabel:`Booking Fees`: charge a fee for booking appointments through the *Calendar* and
+  *Appointment* apps. Requires the :guilabel:`Pay to Book` module to be installed
+- :guilabel:`Combo`: create discounted products sold in a bundle, available with the *POS* app
+  installed
+- :guilabel:`Event Ticket`: sell tickets to attend an event. Requires installation of the *Events*
+  app
+- :guilabel:`Event Booth`: charge customers for setting up a booth. Requires installation of the
+  *Events* app
+- :guilabel:`Course`: sell access to an educational course. Requires installation of the *eLearning*
```

Comment:
Just a note - some of these bullets end with a period, some don't - I think it'd be nice to end them all the same way consistently

Reply:
Thanks for pointing it out!